SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# **Security Vulnerability Management Process for SONiC Community**
+This document outlines SONiC vulnerability reporting and management process. SONiC is a popular choice of cloud providers, enterprises, telecom providers, web service providers and others to build their digital infrastructure.  The security of SONiC is vital for the safety and reliability of our digital transformation. The strong and active cooperation among SONiC community members is key to securing SONiC. This process will be shared through https://github.com/sonic-net/SONiC/SECURITY.md file after TSC approval.  The diagram below illustrates the high-level workflow:
+![](./images/security-process/security.png)
+## 1.Report SONiC Vulnerability
+As a SONiC community member, it is your responsibility to report the discovered vulnerabilities before public disclosure. If you want to report a vulnerability, please encrypt your email and privately send it to security@lists.sonicfoundation.dev. Only SONiC security committee members can access the information in this security mailing list, and they also watch over this mailing list.  When someone reports a new vulnerability, SONiC security committee will assist with the vulnerability assessment, coordinate on the mitigation and fix. If you have a suggested fix or mitigation, please include it in your report. Exploit instruction is very useful and will be kept confidential unless it is already public. SONiC security committee may seek help from SONiC repo maintainers or other domain experts to look into the vulnerability and prepare a mitigation/fix. The collaboration and communication will be private.
+We appreciate security researchers and SONiC users that report vulnerabilities to the SONiC Open Source Community. All reports will be investigated thoroughly by the SONiC security committee. 
+## 2.Vulnerability Disclosure 
+Once a mitigation/fix is reviewed and approved by SONiC security committee, the vulnerability disclosure process starts. 
+Mitigation/fix for public known vulnerabilities will be released right away once approved by SONiC security committee. Fix/mitigation for non-public vulnerabilities should also come out as soon as possible, but we may postpone them if the reporter or an affected party request so. However, the delay should be no more than 90 days from when a mitigation/fix is ready. The SONiC security committee should create a template for disclosing vulnerability.
```

Comment:
It would be good to quantify somewhere in the doc, what comes under  publicly known vulnerability vs (unknown) vulnerability?